modules/swagger-codegen/src/main/resources/python/model.mustache

```diff
@@ -32,96 +37,96 @@ class {{classname}}(object):
                             and the value is json key in definition.
     """
     swagger_types = {
-        {{#vars}}'{{name}}': '{{{datatype}}}'{{#hasMore}},
-        {{/hasMore}}{{/vars}}
+{{#vars}}
+        '{{name}}': {{{datatype}}}{{#hasMore}},{{/hasMore}}
```

Comment:
The `{{{datatype}}}` has to be quoted as it should be a string, otherwise, it generates a broken code.